### PR TITLE
Adding more information to Russian Автомир

### PR DESCRIPTION
**Pull request by bkissin**
Found the wikidata and wikipedia information

### CODE REVIEW

File: brands/shop/car.json [hunk 1]
```diff
@@ -324,6 +324,10 @@
     "tags": {
       "brand": "Автомир",
       "name": "Автомир",
+      "name:en": "Autoworld"
+      "shop": "car"
```
Comment (mds08011) [inline brands/shop/car.json]: This line is not needed as there is already a `"shop": "car"` on line 331.

Comment (bkissin) [reply brands/shop/car.json]: Happy to fix. How do I do it? I'm still new to github.

**Comment by bkissin** [comment]
OK, I think I fixed it.

File: brands/shop/car.json [hunk 2]
```diff
@@ -324,7 +324,10 @@
     "tags": {
       "brand": "Автомир",
       "name": "Автомир",
-      "shop": "car"
+      "name:en": "Autoworld",
+      "shop": "car",
+      "brand:wikidata": "Q4056321",
+      "brand:wikipedia": "ru:Автомир (автодилер)",
```
Comment (mds08011) [inline brands/shop/car.json]: This last line should not have a comma.  I think you'll be set after that.

Comment (bkissin) [reply brands/shop/car.json]: LOL thanks!

**Pull request merged**